script.js

```diff
@@ -1,4 +1,267 @@
+var mode = 'highest'
+var gameMode='dice roll'
+var diceNumberPlayer1=[]
+var diceNumberPlayer2=[]
+var currPlayer=1
+var concastenateNumber1=``
+var concastenateNumber2=``
+var concastenateNumberPlayer1=[]
+var concastenateNumberPlayer2=[]
+var player1NumberWin=0
+var player2NumberWin=0
 var main = function (input) {
```

Comment:
great job being able to extract your logic into helper functions. I think it would make your code even more readable if you place the helper functions outside of the main function so your main function is shorter